config/deployer.sample.json

```diff
@@ -60,7 +60,7 @@
   "AdminEmail": "sysadmin@sample.mattermost.com",
   "AdminUsername": "sysadmin",
   "AdminPassword": "Sys@dmin-sample1",
-  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.9.1/mattermost-load-test-ng-v1.9.1-linux-amd64.tar.gz",
```

Comment:
Is this out of date somehow? Noticing the bump from v1.9 to v1.15.

Reply:
Yeah, this is something we need to fix process-wise. We only bump the version in the release branches, but never on master. We may want to merge into master and then cut the branch? :thinking:

Reply:
> Yeah, that would be my expectation to be honest.

Let's test this now

Reply:
That could also work, at which point the pinning would make sense in the release branch only.